WineCellar/WineCellar.GUI/RegisterWine.xaml

```diff
@@ -37,7 +37,13 @@
                 <StackPanel Orientation="Horizontal" Margin="0 0 0 0">
                     <Label Content="Naam" />
                     <TextBox LostFocus="PlaceholderLostFocus" GotFocus="PlaceholderFocus"  x:Name="name" 
-                             TextWrapping="Wrap" Text="Naam&#xD;&#xA;" Width="200" Height="25" />    
+                             TextWrapping="Wrap" Text="Naam&#xD;&#xA;" Width="200" Height="25" />
```

Comment:
Heeft deze encoded tekst (dat wat in de property "Text" staat) een functie? Zo niet, graag weghalen om leesbaarheid te verbeteren.